docs/_docs/testnet/starting-a-node.md

```diff
@@ -0,0 +1,49 @@
+---
+---
+# Starting a node
+How to start a node in the Evan's Cat testnet.
+
+## Requirements
+- SGX enable VM
+- Docker
+
+## Steps
+#### - Create an SGX enabled VM
+Recommended Standard DC4s v2 (4 vcpus, 16 GiB memory) in azure.
+
+#### - Install Docker
+
+```
+sudo apt-get update \
+    && sudo apt-get install -y jq \
+    && curl -fsSL https://get.docker.com -o get-docker.sh && sh ./get-docker.sh
+```
+
+#### - Download Obscuro Start script
+
+```
+  wget https://raw.githubusercontent.com/obscuronet/go-obscuro/v0.7.1/testnet/start-obscuro-node.sh && \
+  wget https://raw.githubusercontent.com/obscuronet/go-obscuro/v0.7.1/testnet/docker-compose.yml
+```
```

Comment:
Should probably state that the version should be the latest e.g. to navigate to the releases page to get the latest version and then use that. Some people will just copy and paste and end up starting up a v0.7.1 node 6 months from now etc

Reply:
Good point - Updating it.